app-localize-behavior.html

```diff
@@ -167,6 +179,20 @@
     }
   },
 
+  _languageChanged() {
+    // Clear the message cache.
+    var proto = this.constructor.prototype;
+    proto.__localizationCache.messages = {};
+
+    for (const instance of appLocalizeBehaviorInstances) {
+      // Update the language of all instances to match this one.
+      instance.language = this.language;
```

Comment:
Sorry, I didn't mean to push that. I intended for that to be a separate pull request for a separate issue. You can ignore this commit.

I would however like the option to have all instances use the same language. I think that will be common case for most apps.